src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationRecordPattern.java

```diff
@@ -0,0 +1,74 @@
+//non-compiled with javac: Compilable with Java21        //indent:0 exp:0
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;//indent:0 exp:0
+//indent:0 exp:0
+//indent:0 exp:0
+/* Config:                                                                  //indent:0 exp:0
+ * basicOffset = 4                                                          //indent:1 exp:1
+ * braceAdjustment = 0                                                      //indent:1 exp:1
+ * caseIndent = 4                                                           //indent:1 exp:1
+ * tabWidth = 4                                                             //indent:1 exp:1
+ * throwsIndent = 8                                                         //indent:1 exp:1
+ * forceStrictCondition = true                                             //indent:1 exp:1
+ */                                                                         //indent:1 exp:1
+public class InputIndentationRecordPattern {                                //indent:0 exp:0
+    record ColoredPoint(boolean p, int x, int c) { }       //indent:4 exp:4
+    record Rectangle(ColoredPoint upperLeft, ColoredPoint lowerRight) { }        //indent:4 exp:4
```

Comment:
Same. I guess I would expect these 2-3 lines to be next to the end of their code.

I picked out the last 2 since there is more than 1 space to comment, where afterwards there is only a single space to comment, and before they are aligned in same column position.

Reply:
> I am ok with transition from same position to after code, but then there should be some consistency.

I don't get it, please restate how after code and keep consisteny ? 
should all of them be aligned at the same col position ?

Reply:
done the header comments are aligned in the same column position. the code comments are after 1 space of the code